R/doAzureParallel.R

```diff
@@ -525,6 +525,11 @@ setHttpTraffic <- function(value = FALSE) {
     endIndices[length(startIndices)] <- ntasks
   }
 
+  maxTaskRetryCount <- 3
```

Comment:
Should we add some documentation and/or samples on how to use this new parameter? Or else how should user knows there is such an option?

Reply:
I think adding it to the wiki page is sufficient enough